### PR TITLE
Comment change for VXLAN tunnel

### PR DESCRIPTION
**Pull request by MELLANOX-zhichenr**
1. Overlay interface should be bridge port object for VXLAN type.
2. TTL and DSCP mode should be UNIFORM_MODEL (user defined) for VXLAN
   type, regardless of the inner packet type.
3. For VXLAN, it is OK to define ECN_MODE as STARDARD_MODE, while assign
   VNI to VLAN ID mapper to tunnel mapper attribute.

### CODE REVIEW

**Comment by lguohan** [comment]
it has conflicts with current master, can you resolve the conflicts?

**Comment by MELLANOX-zhichenr** [comment]
Conflict resolved.

File: inc/saitunnel.h [hunk 1]
```diff
@@ -229,7 +229,8 @@ typedef enum _sai_tunnel_attr_t
     SAI_TUNNEL_ATTR_UNDERLAY_INTERFACE,
 
     /** tunnel overlay interafce [sai_object_id_t] (MANDATORY_ON_CREATE|CREATE_ONLY)
-     *  overlay interface is router interface when SAI_TUNNEL_ATTR_TYPE = SAI_TUNNEL_IPINIP or SAI_TUNNEL_IPINIP_GRE */
+     *  overlay interface is router interface when SAI_TUNNEL_ATTR_TYPE = SAI_TUNNEL_IPINIP or SAI_TUNNEL_IPINIP_GRE
+     *  overlay interface is bridge port object when SAI_TUNNEL_ATTR_TYPE = SAI_TUNNEL_VXLAN */
```
Comment (lguohan) [inline inc/saitunnel.h]: meeting minutes: in dot.1d use case, the overlay interface is derived from vni to bridge port mapping. this is only used for dot1q, use case.

**Comment by lguohan** [comment]
is this one still needed?

**Comment by MELLANOX-zhichenr** [comment]
This pull request is out-dated, since <bridge_port_interface, vni> map is introduced for vxlan.

**Pull request closed without merging**